packages/rspeedy/core/src/plugins/dev.plugin.ts

```diff
@@ -163,6 +164,16 @@ export function pluginDev(
                   ),
                   'default'
                 ],
+              } : {
```

Comment:
Instead of writting `__webpack_dev_server_client__` twice, can we use another `ProvidePlugin` for `isLynx` to provide `WebSocket`?